CMakeLists.txt

```diff
@@ -178,6 +164,4 @@ if(CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
 
     include(CTest)
     add_subdirectory(tests)
-else()
-    set_target_properties(otf2xx-reader PROPERTIES EXCLUDE_FROM_ALL TRUE)
```

Comment:
This ensures that the reader library is only built when the application is actually using it. The writer interface is header only, no need to build the reader.